src/com/owncloud/android/authentication/AuthenticatorActivity.java

```diff
@@ -1879,9 +1879,6 @@ private void showUntrustedCertDialog(RemoteOperationResult result) {
 
     }

```

Comment:
Update the year and include your name as @author at the top of the class

Reply:
done